export_presets.cfg

```diff
@@ -83,6 +85,7 @@ debug/export_console_wrapper=1
 binary_format/embed_pck=false
 texture_format/s3tc_bptc=true
 texture_format/etc2_astc=false
+shader_baker/enabled=false
```

Comment:
I wonder if enabling shader baker would help with the freeze the first time an ink blob appears https://github.com/endlessm/threadbare/issues/444 . Although there are tradeoffs https://godotengine.org/article/dev-snapshot-godot-4-5-dev-5/#shader-baker